frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Form, FormTitle, Icon, TitleInput, MessageInput, SubmitInput } from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const [postObject, setPostObject] = useState({});
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setPostObject((values) => ({ ...values, [name]: value }));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(postObject);
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <FormTitle>Add new post</FormTitle>
+      <TitleInput
+        type="text"
+        name="title"
+        placeholder="Enter title..."
+        value={postObject.title || ''}
```

Comment:
```suggestion
        value={postObject.title}
```
A jak zrobisz coś takiego, to źle będzie działać?

Reply:
Będzie działać dobrze, ale z warningiem w konsoli.

Reply:
No to nie, warningow nie chcemy :)

Reply:
Tak też myślałem ;)